### ISSUE TITLE
[RE-TEST] Modal's close icon has a partially hidden focus indicator on iPhone

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#6666]
Tested on iPhone 7

![Screen Shot 2020-02-19 at 11 34 05 PM](https://user-images.githubusercontent.com/15113219/75007064-ed5b1c00-5441-11ea-8a56-53e2224c76f9.png)

**Comment by tobi-or-not-tobi** [comment]
should be fixed as part of #6581

**Comment by giancorderoortiz** [comment]
Issue has been fixed.
Retest with 3.0 and 3.4

(3.0)
Linux: https://spartacus-demo.eastus.cloudapp.azure.com/electronics-spa/en/USD/

(3.4)
https://spartacusstore.cg79x9wuu9-eccommerc1-p5-public.model-t.myhybris.cloud/electronics-spa/en/USD/

Used iPhone7, Safari, Chrome

![IMG_6501](https://user-images.githubusercontent.com/46171897/122681461-6dbe9800-d1c2-11eb-82f4-6fa4d9f4a8a5.PNG)
![IMG_6502](https://user-images.githubusercontent.com/46171897/122681462-6e572e80-d1c2-11eb-9cf3-fc4db0df773c.PNG)
